src/components/TaskPreviewPanel.js

```diff
@@ -1,68 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
```

Comment:
Since TaskPreviewPanel is a functional component, we should stick to the convention and use hooks instead of withTheme and withStyles.
```
import { makeStyles } from '@material-ui/styles';
const useStyles = makeStyles((theme) => ({
STYLES_HERE
}));
const classes = useStyles();
```

Reply:
done